WearTilesKotlin/app/src/main/java/com/example/wear/tiles/tools/WearPreviewDevices.kt

```diff
@@ -15,34 +15,11 @@
  */
 package com.example.wear.tiles.tools
 
-import androidx.compose.ui.tooling.preview.Devices
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.wear.tooling.preview.devices.WearDevices
 
 @Preview(
-    device = Devices.WEAR_OS_LARGE_ROUND,
-    showSystemUi = true,
-    backgroundColor = 0xff000000,
-    showBackground = true,
-    group = "Devices - Large Round"
-)
-@Preview(
-    device = Devices.WEAR_OS_SMALL_ROUND,
-    showSystemUi = true,
-    backgroundColor = 0xff000000,
-    showBackground = true,
-    group = "Devices - Small Round"
-)
-@Preview(
-    device = Devices.WEAR_OS_SQUARE,
-    showSystemUi = true,
-    backgroundColor = 0xff000000,
-    showBackground = true,
-    group = "Devices - Square"
-)
-public annotation class WearPreviewDevices
-
-@Preview(
-    device = Devices.WEAR_OS_SMALL_ROUND,
+    device = WearDevices.SMALL_ROUND,
```

Comment:
We have these from Wear Compose now.


    implementation("androidx.wear.compose:compose-ui-tooling:1.2.0")